### ISSUE TITLE
Parallel instances

### ISSUE THREAD

**Issue by m0nkge** [unoconv/unoconv#580]
When running unoconv in a loop as follows everything works OK but its slow.

``for i in *.odt ; do R=`unoconv --stdout -f text  $i | grep -w "word" -c` ; echo $R $i; done``

Attempting to speed it up, I spawned each iteration in the background. 

```
LOG="/tmp/odtresults-`date  +'%Y%m%d%H%M%S'`.txt"
for i in *.odt ; do
        (
                R=`unoconv --stdout -f text  $i | grep -w -o "word" | wc -l`
                echo $R $i >>$LOG
        ) &
done
```

but this doesnt work as expected. I get a bunch of 

`Error: Unable to connect or start own listener. Aborting.`

errors and a whole lot of stuck processes.

How can I get this to work?

Cheers.

**Comment by m0nkge** [comment]
Ok, I saw your note about starting the listener beforehand and I used your example.

how adding 
```
unoconv -l &
sleep 15s
```

fixes my problem. But I'd like to eliminate the 15s delay if possible. How can I know when the listener is ready?

**Comment by regebro** [comment]
I don't think you can. But a 15 second delay seems excessive? It's just a question of LibreOffice to start, does that take 15 seconds for you?

**Comment by regebro** [comment]
No response.
